cmake/assets/run_nodered_template.sh.in

```diff
@@ -1,3 +1,17 @@
-cd @EVEREST_DOCKER_DIR@
-docker build -t everest-nodered .
-docker run --rm --network host --name everest_nodered --mount type=bind,source=@FLOW_FILE@,target=/data/flows.json everest-nodered
+# In the following a volume is created to contain the nodered config, this is done to
+# allow starting the nodered container from inside a devcontainer
```

Comment:
Why does using a volume allow to start the node-red container from inside a dev container?

Reply:
The workspace inside the devcontainer is mounted to an other path. So you would need to know about the host path of the flow file.

`docker run` is executed on host with host paths

`docker cp`  is executed with container's paths 

This way it is possible to copy a file with only knowing the devcontainer paths into a docker volume which then can be mounted by name into the nodered container

Reply:
I see, well this is really ugly.  These docker inside docker scenarios are quite special, not sure whether we should support these - they might break quite easily.  This script here basically only works, if the devcontainer has been launched correctly - which is an assumption I wouldn't necessarily make.
Does this script work, when executed from the _host_ (i.e. outside of the container?)

Reply:
It is not real docker inside docker, because it is more like remote controlling the hosts docker socket.
This script is designed to work inside and outside the devcontainer. Of course it needs to be rebuild when changed because of paths